### ISSUE TITLE
The number of epoch

### ISSUE THREAD

**Issue by Jee-King** [phoenix104104/LapSRN#6]
How many epoch can be trained to achieve the effect of the paper？
Thank you for your answer.

**Comment by ZJiaQi** [comment]
Hi, thanks for your code, but I did not find the part of robust loss function. Can you show me the part of definnition? Thank you so much.
